ergo-lib/src/wallet/mnemonic.rs

```diff
@@ -1,8 +1,19 @@
 //! Mnemonic operations according to BIP32/BIP39
 
+#[cfg(feature = "mnemonic_gen")]
```

Comment:
I'd rather have `mnemonic_gen` feature attribute on the whole module, to avoid putting on almost every statement. `Mnemonic::to_seed` could be extracted into a function and moved out.